content/docs/user-guide/data-pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,237 @@
+# Defining Pipelines
+
+DVC pipelines are written as collections of data processing <abbr>stages</abbr>
+in `dvc.yaml` files. These represent data workflows that you want to
+**reproduce** reliably later. Here's a sample 3-stage structure:
+
+```yaml
+stages:
+  prepare: ... # stage 1 definition
+  train: ... # stage 2 definition
+  evaluate: ... # stage 3 definition
+```
+
+<admon>
+
+This _codification_ has the added benefit of allowing you to develop pipelines
```

Comment:
I doubt this will be understood. Any pipelines I've seen so far are described in one way or another as code.

Reply:
Hmm. I think the "codification" part is relatively easy to get. The GitOps part is what comes out of nowhere, it should probably be a whole other section or part of a future page ⏳